mesonbuild/interpreter/interpreter.py

```diff
@@ -2307,10 +2307,9 @@ def func_install_man(self, node: mparser.BaseNode,
         KwargInfo('install_tag', (str, NoneType), since='0.62.0')
     )
     def func_install_emptydir(self, node: mparser.BaseNode, args: T.Tuple[str], kwargs) -> None:
-        d = build.EmptyDir(args[0], kwargs['install_mode'], self.subproject, kwargs['install_tag'])
-        self.build.emptydir.append(d)
-
-        return d
+        d = [build.EmptyDir(one_dir, kwargs['install_mode'], self.subproject, kwargs['install_tag'])
+             for one_dir in args]
```

Comment:
We need to do something here to say that a accepting an array of arguments is a new feature. I would suggest something like:
```python
if`len(args) > 1:
    FeatureNew.single_use(...)
```

Reply:
Done!